styles.css

```diff
@@ -20,11 +20,40 @@ body {
   overflow-x: hidden;
 }
 
-header{  
+header {
   display: flex;
   margin-left: 5%;
 }
 
+.text_arrow img {
+  margin-left: 320%;
+}
+
+.text_arrow div {
+  margin-left: 331%;
+}
```

Comment:
- [OPTIONAL] Kindly compare your design with the figma design and work your design alongside figma design

### Your Design
![your-framework](https://user-images.githubusercontent.com/26597432/221378691-22ede63a-009a-40b6-b8a0-5c6c1c832374.png)

### Figma Design 
![temp-framework](https://user-images.githubusercontent.com/26597432/221378698-2cb3a017-2c47-42f8-9008-302c398bce4b.png)